pkg/proxy/userspace/proxier.go

```diff
@@ -231,7 +234,11 @@ func NewCustomProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptab
 
 	err = setRLimit(64 * 1000)
 	if err != nil {
-		return nil, fmt.Errorf("failed to set open file handler limit: %v", err)
+		if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.KubeletInUserNamespace) && libcontaineruserns.RunningInUserNS() {
+			klog.ErrorS(err, "failed to set open file handler limit to 64000 (running in UserNS, ignoring)")
```

Comment:
In the other files, `(running in UserNS, ignoring)` is emitted using `V(2).InfoS`. Is `ErrorS` intentional here?

Reply:
fixed